Update contribution guidelines (#91)
esa/torchquad: README.md

      conda activate torchquad
      ```

Once the installation is done, then you are ready to contribute. 

> Please note that PRs should be created on the `develop` branch.

Reviewer: ```suggestion
Please note that PRs should be created from and into the `develop` branch. For each release the develop branch is moved into main.
```

3. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
4. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
5. Push to the Branch (`git push origin feature/AmazingFeature`)

> 6. Open a Pull Request on the `develop` branch (*not* `main`)

Reviewer: ```suggestion
6. Open a Pull Request on the `develop` branch (*not* `main`) . ( NB For torchquad, we autoformat every PR with black. Our GitHub actions may create additional commits on your PR for that reason.)
```

and we will have a look at your contribution as soon as we can. 

> For torchquad, we autoformat every PR with Black. This means that if you are not using Black, you will have to pull the autoformatted code after pushing, otherwise conflicts might arise.

Reviewer: ```suggestion
```

> Furthermore, we ask you to create PRs on the `develop` branch, *not* `main` branch, and to make sure that your PR has passed all tests.

Reviewer: ```suggestion
Furthermore, please make sure that your PR passes all automated tests. Review will only happen after that.
```

> Only PRs created on the `develop` branch with all tests passed will be considered.

Reviewer: ```suggestion
Only PRs created on the `develop` branch with all tests passed will be considered. The only exception to this rule is if you want to update the documentation in relation to the current release on conda / pip. In that case you may ask to merge directly into `main`.
```
